tests/data/format/linewrap_summary/function_docstrings.args

```diff
@@ -1 +1,2 @@
 --linewrap-full-docstring
+--max-summary-lines=2
```

Comment:
Same as above, where possible I think the code should be updated instead of the arguments. That makes the test more easily "interpretable". You can assume that all default values are being used instead of having to check every `.args` file to see which specific configuration each test is using.

Reply:
this one can be done